api/src/pcapi/core/mails/transactional/bookings/booking_finance_incident_notification.py

```diff
@@ -0,0 +1,23 @@
+from pcapi.core import mails
+from pcapi.core.finance import models as finance_models
+from pcapi.core.mails import models as mails_models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+
+
+def send_individual_finance_incident_email(
+    finance_incident: finance_models.FinanceIncident,
+) -> bool:
+    offer = finance_incident.booking_finance_incidents[0].booking.stock.offer
+    venue = finance_incident.venue
+    data = mails_models.TransactionalEmailData(
+        template=TransactionalEmail.RETRIEVE_INCIDENT_AMOUNT_ON_INDIVIDUAL_BOOKINGS.value,
+        params={"OFFER_NAME": offer.name, "VENUE_NAME": venue.publicName or venue.name},
```

Comment:
Il y a déjà une propriété pour le nom du lieu : 
```suggestion
        params={"OFFER_NAME": offer.name, "VENUE_NAME": venue.common_name},
```